lib/Auth/Process/COmanageDbClient.php

```diff
@@ -950,17 +1122,16 @@ private function getCouRootParent($cou_name, $cou_nested) {
      * @return bool
      */
     private function isRootCou($couid, $cou_nested) {
-      foreach($cou_nested as $hierarchy) {
-        $root_key = array_keys($hierarchy['path_full_list'])[0];
-        if($root_key == $couid) {
-          return true;
+        foreach ($cou_nested as $hierarchy) {
+            $root_key = array_keys($hierarchy['path_full_list'])[0];
+            if($root_key == $couid) {
+                return true;
+            }
         }
-      }
-      return false;
+        return false;
     }
 
-    private function _showException($e)
-    {
+    private function _showException($e) {
```

Comment:
Opening braces for methods MUST go on the next line, and closing braces MUST go on the next line after the body. (see https://www.php-fig.org/psr/psr-2/#1-overview)